modules/@apostrophecms/page/ui/apos/components/AposPagesManager.vue

```diff
@@ -221,6 +225,8 @@ export default {
     // Get the data. This will be more complex in actuality.
     this.modal.active = true;
     await this.getPages();
+    this.modal.triggerFocusRefresh++;
```

Comment:
take into account retrieved pages elements to add them in the list of elements to focus.